### ISSUE TITLE
No clear path to handle Accessibility pan gestures

### ISSUE THREAD

**Issue by freerangenerd** [facebook/react-native#20478]
I have a pure TypeScript custom slider component with a PanResponder for capturing events to adjust the slider. It works as expected when the screen reader is not active, but I am unable to make it work properly with it enabled.

React Native Accessibility docs provide APIs for setting an accessibility label and some OS specific traits to indicate to the user information about the control, but I don't see a means to handle the gestures required. For example, on iOS setting the trait 'adjustable' causes VoiceOver to tell the user to "Swipe up and down" to change values. PanResponder doesn't seem to have any kind of event mapping to make the control actually respond that way.

Is there a way to make this work?
